src/Tribe/Main.php

```diff
@@ -291,7 +291,7 @@ public function before_html_data_wrapper( $html ) {
 
 			$data_attributes = array(
 				'live_ajax'         => tribe_get_option( 'liveFiltersUpdate', true ) ? 1 : 0,
-				'datepicker_format' => tribe_get_option( 'datepickerFormat' ),
+				'datepickerFormat'  => tribe_get_option( 'datepickerFormat' ),
```

Comment:
I must be missing something obvious - how would `date-datepicker_format` cause a bug?

I can say with confidence that uppercase letters are not desirable in attribute names [according to W3C](http://www.w3.org/html/wg/drafts/html/master/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes).

> A custom data attribute is an attribute in no namespace whose name starts with the string "data-", has at least one character after the hyphen, is XML-compatible, and contains __no uppercase ASCII letters__.